### DISCUSSION TITLE
tine podcast: 16 - Abschied von der Community Edition

### DISCUSSION THREAD

**Discussion by pschuele** [tine-groupware/tine#4] [Announcements]
https://www.tine-groupware.de/podcast/16-abschied-von-der-community-edition/

Herzlich willkommen zum tine Podcast! Wir möchten Euch zu einem Blick hinter die Kulissen der Open-Source-Groupware tine mitnehmen. Achtung! Manchmal kann es technisch werden.

Seit der Veröffentlichung von Ellie erreichten uns einige Fragen zur Community Edition von tine. Ihr müsst jetzt ganz stark sein – die Community Edition in ihrer bisherigen Form gibt es nicht mehr. Dafür gibt es andere Wege und ein neues github-Angebot. Über das Warum und was das für die Community bedeutet, reden wir in dieser Episode mit Philipp Schüle, Senior Core Entwickler und github-Verantwortlicher, und Cornelius Weiss, Teamleiter Software Engineering.

Gebt uns gerne Feedback zu dieser Episode. Habt Ihr vielleicht Themen, über die wir unbedingt mal sprechen sollten oder andere Anregungen? Gern auch hier in den Discussions oder hier: https://www.tine-groupware.de/kontakt/. Wenn es Euch gefallen hat, hinterlasst uns doch eine positive Bewertung im Podcast-Portal Eurer Wahl.

Neue Episoden erscheinen immer am vierten Mittwoch des Monats – am besten richtet Ihr Euch schon mal einen Serientermin in tine ein.

**Comment by sbaumgartner75** [comment]
Ab wann gibt's denn die "weeklies" wieder und wo?

**Reply by pschuele** [reply]
wir sind dran, denke bis ende Januar sollten sie wieder verfügbar sein.

**Reply by pschuele** [reply]
weekly releases sind seit letztem SA wieder verfügbar: https://github.com/tine-groupware/tine/releases?q=weekly&expanded=true

**Comment by JPRuehmann** [comment]
Wie komme ich auf die Weeklies? das aktuelste was ich nach mehreren Tagen ans laufen gebracht habe ist die 2023.11.6.

**Reply by pschuele** [reply]
https://github.com/tine-groupware/tine/releases?q=weekly
